src/docs/admin/ProActiveAdminGuide.adoc

```diff
@@ -1462,6 +1462,25 @@ that automatically adds indexes on foreign keys, it is strongly recommended to
 ask your DBA to identify and delete duplicate indexes created by the database
 system since they may hurt performances.
 
+==== Housekeeping
+
+The Scheduler provides a housekeeping mechanism that periodically removes finished jobs
+from the Scheduler Portal. It can also remove jobs and all its data from the database to save space.
+This mechanism has two phases:
+
+. Once a job is finished, set its scheduled time for removal (the job expiration date)
+. Periodically trigger the cleaning of the expired jobs off the scheduler and/or the database
```

Comment:
I would rephrase for better understanding. "Actual cleaning of expired jobs from the scheduler and/or the database will be periodically triggered and performed in a bulk operation."